tests/console/ovs_server.pm

```diff
@@ -0,0 +1,141 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test IPsec tunnel in Open vSwitch with 3 different authentication methods
+#
+#   This test does the following
+#    - Installs openvswitch-ipsec and openvswitch-pki
+#    - Starts the systemd service unit
+#    - Executes IPsec tunneling between two hosts with the following authenitcation
+#    methods:
+#    	* Pre-shared key
+#    	* Self-signed certificate
+#    	* CA-signed certificate
+#    - This host verifies that IPsec GRE tunnel is running between the two hosts
+#
+# Maintainer: Anna Minou <anminou@suse.de>
+#
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+use console::ovs_utils;
+
+my $server_ip   = "10.0.2.101";
+my $client_ip   = "10.0.2.102";
+my $id          = "1";
```

Comment:
Can you define something like `$server_vpn` and `$client_vpn` here?
I saw `192.0.0.2` and I didn't know where it came from.

Reply:
yes, you're right